src/olympia/users/templates/users/edit.html

```diff
@@ -92,6 +92,7 @@
               <img src="{{ amouser.picture_url }}"
                    alt="{% if not amouser.picture_type %}{{ _('No Photo') }}{% endif %}"
                    class="avatar photo">
+              <br>
```

Comment:
You should be able to add `clear: both` to `.invisible-upload` in CSS to get it to clear without adding the line break.